label.py

```diff
@@ -20,8 +21,12 @@
 if args.verify:
     images_to_show = [i for i in utils.get_images() if i in labels]
 else:
-    images_to_show = [i for i in utils.get_images() if i not in labels]
-    random.shuffle(images_to_show)
+    all_labels = utils.read_labels()
+    unlabelled_in_all_labels = [i for i in utils.get_images() if i in all_labels and i not in labels]
```

Comment:
Use images_to_show here and for `unlabelled`, so you don't have to call get_images() twice.